### PR TITLE
WF-63099-Update Presentation control UG documentation with tagged PDF code snippet

### PR DESCRIPTION
**Pull request by AntonihilSahayaraj**
<p>Kindly review the below merge request and let me know your feedback.</p>
<table>
<tbody>
<tr>
<td>
<p>Task Link</p>
</td>
<td>
<p><a href="https://syncfusion.atlassian.net/browse/WF-63099">https://syncfusion.atlassian.net/browse/WF-63099</a></p>
</td>
</tr>
<tr>
<td>
<p>Description</p>
</td>
<td>
<p>Update Presentation control UG documentation with tagged PDF code snippet</p>
</td>
</tr>
<tr>
<td>
<p>Solution</p>
</td>
<td>
<p>Add a code snippet in UG page to convert PowerPoint Presentation to PDF with structures tags.</p>
</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 1]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+
+//Load the PowerPoint presentation into stream.
+using (FileStream fileStreamInput = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read))
+{
+    //Open the existing PowerPoint presentation with loaded stream.
+    using (IPresentation pptxDoc = Presentation.Open(fileStreamInput))
+    {
+        //Instantiation of PresentationToPdfConverterSettings.
+        PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+        // Enable a flag to preserve document structured tags in the converted PDF document.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1385]: // Enable a flag to preserve structured document tags in the converted PDF document.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 2]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+using (FileStream fileStreamInput = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read))
+{
+    //Open the existing PowerPoint presentation with loaded stream.
+    using (IPresentation pptxDoc = Presentation.Open(fileStreamInput))
+    {
+        //Instantiation of PresentationToPdfConverterSettings.
+        PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+        // Enable a flag to preserve document structured tags in the converted PDF document.               
+        pdfConverterSettings.AutoTag = true;
+        //Convert the PowerPoint document to PDF document.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1387]: //Convert the PowerPoint document to a PDF document.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 3]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+        using (PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings))
+        {
+            //Save the converted PDF document to fileStream.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1390]: //Save the converted PDF document to the fileStream.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 4]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+            }
+        }   
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+
+//"App" is the class of Portable project.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1404]: //The "App" is the class of the Portable project.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 5]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open a PowerPoint presentation.
+using (IPresentation pptxDoc = Presentation.Open(assembly.GetManifestResourceStream("GettingStarted.Assets.Sample.pptx")))
+{
+    //Instantiation of PresentationToPdfConverterSettings.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1409]: //Instantiation of the PresentationToPdfConverterSettings.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 6]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    // Enable a flag to preserve document structured tags in the converted PDF document.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1411]: // Enable a flag to preserve structured document tags in the converted PDF document.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 7]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open a PowerPoint presentation.
+using (IPresentation pptxDoc = Presentation.Open(assembly.GetManifestResourceStream("GettingStarted.Assets.Sample.pptx")))
+{
+    //Instantiation of PresentationToPdfConverterSettings.
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    // Enable a flag to preserve document structured tags in the converted PDF document.               
+    pdfConverterSettings.AutoTag = true;
+    //Convert the PowerPoint Presentation into PDF document.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1413]: //Convert the PowerPoint Presentation into a PDF document.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 8]
```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
...
+    using (PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings))
+    {
+        //Save the converted PDF document to MemoryStream.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1416]: //Save the converted PDF document to the MemoryStream.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**